Assets/Scripts/Scriptable Objects/SlamBehaviour.cs

```diff
@@ -0,0 +1,71 @@
+using UnityEngine;
+using UnityEngine.AI;
+
+[CreateAssetMenu(fileName = "Slam Behaviour", menuName = "Behaviours/Slam")]
+public class SlamBehaviour : AIBehaviour
+{
+    public float range = 10f;
+    public float radius = 2f;
+    public int damage = 1;
+    public float cooldown = 5f;
```

Comment:
Doesn't need to be public 
```suggestion
    [SerializeField] private float range = 10f;
    [SerializeField] private float radius = 2f;
    [SerializeField] private int damage = 1;
    [SerializeField] private float cooldown = 5f;
```